src/main/config/config.c

```diff
@@ -36,6 +36,9 @@
 #include "config/config_eeprom.h"
 #include "config/feature.h"
 
+#ifdef USE_TELEMETRY_CASTLE
+#include "drivers/castle_telemetry_decode.h"
+#endif
```

Comment:
The convention in BF and RF is to include .h files unconditionally.
If there are things that should not be included, then the ifdef should be in the .h file itself.
Most of the time the headers can be included without side effects. It also guarantees that the rest of the code won't broke if the flag is defined/undefined.

Reply:
done.